bff/hosts/Hosts.AppHost/Program.cs

```diff
@@ -49,8 +50,13 @@
     .WithReference(bffBlazorPerComponent)
     .WithReference(bffBlazorWebAssembly)
     .WithReference(apiDPop)
-    .WithReference(bffDPop)
-    ;
+    .WithReference(bffDPop);
+
+builder.AddProject<BffLocalApi>("templates-bff-local", launchProfileName: null)
+    .WithHttpsEndpoint(5300, name: "bff-local");
+
+builder.AddProject<BffRemoteApi>("templates-bff-remote", launchProfileName: null)
+    .WithHttpsEndpoint(5310, name: "bff-remote");
```

Comment:
Really smart to add the templates themselves to aspire. We need to get aspire going in the same way on the IdentityServer side! Do we have automated tests that sanity check all the aspire projects? Should we add the templates to that?

Reply:
that is my plan, but one step at a time